### PR TITLE
[READY] jagged spikes, disruptor blades, nullrod bonus damage fix, armblade inhands fix

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
jagged spikes:
- printable from a hacked lathe, can get a bandolier from stolen supply crates
- 100% embed chance, tiny, 25 force, bonus damage when embedded in a human

disruptor blades:
- unobtainable
- bonus damage against animals and aberrations

null rods:
- now actually do bonus damage against demonic and aberrations
## Why It's Good For The Game
jagged spikes: well sometimes you just need violence
disruptor blades: i forgot to clean my branch but w/e they're not implemented
null rods: bugfix
## Changelog
:cl:
fix: Null rods and equivalents now actually do the bonus damage they're intended to do against demonic creatures and aberrations.
add: Jagged spikes, which are good for jamming into people's ribs and then being guilty of a crime because that's assault with a deadly stick, are now printable from an autolathe. You can't keep your grip on them for long, though.
add: Disruptor blades, which are intended to deal bonus damage to animals and aberrations, are in the code, but unobtainable.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/swords_axes_etc.dm [hunk 1]
```diff
@@ -106,3 +106,49 @@
...
+	sharp = TRUE
+	edge = TRUE
+	icon_state = "embed_spike"
+	item_icons = list(
+			slot_l_hand_str = 'icons/mob/items/lefthand_material.dmi',
+			slot_r_hand_str = 'icons/mob/items/righthand_material.dmi',
+			)
+	item_state = "switchblade_open"
+	sharp = TRUE
+	edge = TRUE
```
Comment (Arturlang) [inline code/game/objects/items/weapons/swords_axes_etc.dm:154]: double sharp and edge here

Comment (Hatterhat) [reply code/game/objects/items/weapons/swords_axes_etc.dm:154]: yes that's intentional this fucking codebase is awful with how it handles pointy things

Comment (Hatterhat) [reply code/game/objects/items/weapons/swords_axes_etc.dm:154]: oh wait fuck i see it now. still this codebase sucks

**Comment by Captain277** [comment]
Yo 2hats. This has a WIP tag but it also says [READY] in the title. You good for a fullmerge or do you want to give it more time to test?

**Comment by Hatterhat** [comment]
the wip was for the initial opening now it's probably fine to merge

**Pull request merged**